cpp/include/cuspatial/experimental/point_polygon_distance.cuh

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <rmm/cuda_stream_view.hpp>
+
+namespace cuspatial {
+
+/**
+ * @ingroup distance
+ * @brief Computes pairwise multipoint to multipolygon distance
+ *
+ * @tparam MultiPointRange An instance of template type `MultiPointRange`
+ * @tparam MultiPolygonRange An instance of template type `MultiPolygonRange`
```

Comment:
These docs are a bit self-referential!

Reply:
This is meant to be `multipoint_range` and `multipolygon_range` (lower case), referring to the name of the templates.